Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:18 as builder
+
+COPY ./ /build/
+
+WORKDIR /build
+
+RUN npm install -g pnpm && \
+    pnpm install && \
+    pnpm run build
+
+FROM node:18-alpine as production
+
+WORKDIR /app
+ARG GIT_SHA
+ENV GIT_SHA=${GIT_SHA}
+ENV NITRO_HOST=0.0.0.0
```

Comment:
This host is used by nuxt? or needs to also be used in the command?

Reply:
You only need these things for deploying to a staging or production environment.